### PR TITLE
Translate : docs/general/implementation.md

### PR DESCRIPTION
**Pull request by neoseurae12**
Related issue: https://github.com/Azure/azure-sdk-korean/issues/121

translation to be continued..

### CODE REVIEW

**Comment by neoseurae12** [comment]
@microsoft-github-policy-service agree

**Comment by ianychoi** [comment]
Thanks - I will proceed to review this PR once all the HTML comment ( `<!--` and `-->` ) parts are removed

File: docs/general/implementation.md [hunk 1]
```diff
@@ -133,55 +133,61 @@ HTTP 파이프라인은 여러 정책에 의해 감싸지는 HTTP 전송으로 
...
+{% include requirement/MUST id="general-logging-no-sensitive-info" %} 승인된 헤더 및 쿼리 매개변수 중 서비스에서 제공하는 "허용 목록(allow-list)"에 있는 헤더 및 쿼리 매개변수만 로깅하세요. 다른 모든 헤더 및 쿼리 매개 변수에 대해서는 해당 값들이 편집(redacted)되어야 합니다.
 
-{% include requirement/MUST id="general-logging-requests" %} log request line and headers as an `Informational` message. The log should include the following information:
+{% include requirement/MUST id="general-logging-requests" %} 요청 라인과 헤더를 `Informational` 메시지로 기록하세요. 로그에는 다음 정보가 포함되어야 합니다:
+
+* HTTP 메서드.
+* URL.
+* 쿼리 매개변수 (허용 목록에 없을 경우 삭제).
+* 요청 헤더 (허용 목록에 없을 경우 삭제).
+* 상관관계 목적으로 SDK에서 제공한 요청 ID.
```
Comment (neoseurae12) [inline docs/general/implementation.md:167]: 검토 필요한 번역 사항
"An SDK provided request ID for correlation purposes."
'correlation purposes'를 '상관관계 목적으로'로 번역해도 자연스러운가? 
'상관관계 목적'이란 어떤 구체적인 목적을 말하는가?

File: docs/general/implementation.md [hunk 2]
```diff
@@ -65,79 +50,39 @@ YOU MAY use client library-specific environment variables for portal-configured
...
-use non-alpha-numeric characters in your environment variable names with the exception of underscore. This ensures broad interoperability.
- -->
+
 
 ## 매개변수 유효성 검사
-<!--
-Parameter validation 
- -->
 
 서비스 클라이언트에는 서비스에 대한 요청을 수행하는 여러 방법들이 있을 것입니다. _서비스 매개변수_들은 유선을 통해 Azure 서비스로 직접 전달됩니다. _클라이언트 매개변수_는 서비스로 직접 전달되지 않고, 클라이언트 라이브러리 내에서 요청을 이행하는 데 사용됩니다. 클라이언트 매개변수의 예로는 URI를 구성하는 데 사용되는 값이나, 스토리지에 업로드해야 하는 파일이 있습니다.
```
Comment (neoseurae12) [inline docs/general/implementation.md:65]: 검토 필요한 번역 사항
"across the wire" => 정말 "유선을 통해"라는 의미인가?

**Comment by neoseurae12** [comment]
> every HTML comment (  ) part is removed :)

**Comment by neoseurae12** [comment]
> Thanks - I will proceed to review this PR once all the HTML comment ( `<!--` and `-->` ) parts are removed

every HTML comment ( ) part is removed :)

**Comment by ianychoi** [comment]
Thank you - I am merging it now but please see my comment to reflect this through another PR thanks!

**Pull request merged**